addons/website/static/tests/tours/snippets_all_drag_and_drop.js

```diff
@@ -41,6 +41,7 @@ for (const snippet of snippetsNames) {
             trigger: ".modal-footer .btn-secondary",
         });
     } else if (['s_popup', 's_newsletter_subscribe_popup'].includes(snippet)) {
+        snippetSteps[1]['in_modal'] = false;
```

Comment:
How did it work before?

Reply:
Before, the modal was shown by the public widget after a delay, when stopping the drag and drop. Now it is necessary, as the modal is shown when dragging over a dropzone